packages/modelviewer.dev/examples/lighthouse.html

```diff
@@ -74,19 +74,32 @@
 
     .progress-bar {
       display: block;
-      width: 0%;
+      width: 33%;
       height: 10%;
+      max-height: 2%;
```

Comment:
Does height 10% mean much when max-height is 2%?

Reply:
The height has to be specified, given the setup. If you play with [this](https://www.bitdegree.org/learn/best-code-editor/css-height-and-width-dimensions-example-5) - Comment out min-height and height, then the max-height doesn't do anything. As long as height is set, though, max-height will override it.

Reply:
10% was an arbitrary number